### PR TITLE
adds spanish translation and string translation for other strings in app

### PR DESCRIPTION
**Pull request by thehme**
- creates translation for existing strings
- takes out other hardcoded strings and creates translations for these and replaces string variables using `context`

### CODE REVIEW

**Comment by DeeDeeG** [comment]
Tests pass at Travis CI :tada: https://travis-ci.com/RefugeRestrooms/refugerestrooms-android/builds/129792911

Someone other than myself will have to look at this  for the technical changes, since I'm not an Android/Java code person. I can help coordinate with someone to review the translations and give any feedback.

If this looks technically okay, and the translations are approved, then we can merge.

File: app/src/main/res/values-es-rUS/strings.xml [hunk 1]
```diff
@@ -0,0 +1,71 @@
+<?xml version="1.0" encoding="utf-8"?>
+<resources>
+    <string name="app_name">Baños para Refugiados</string>
```
Comment (DeeDeeG) [inline app/src/main/res/values-es-rUS/strings.xml]: > Baños para Refugiados

If I recall correctly, we are treating "Refuge Restrooms" as being like a brand name, and we mostly lean toward not translating it on the web app. IMO we can leave it untranslated here as well.

Previous discussion in the comments of this Pull Request: https://github.com/RefugeRestrooms/refugerestrooms/pull/427#issuecomment-362434981

**Comment by DeeDeeG** [comment]
Hi @Intimaria, I wanted to let you know there has been a Spanish translation submitted for Refuge Restrooms' Android app!

If you would be able to review the translations, that would be greatly appreciated! (Also, if not that is totally okay too!)

If you would like to help review the translations, you can see them here: [`values-es-rUS/strings.xml`](https://github.com/RefugeRestrooms/refugerestrooms-android/pull/77/files#diff-2b5d718e97db44f62e321bf20c2363d2)

And compare with the original English words here: [`values/strings.xml`](https://github.com/RefugeRestrooms/refugerestrooms-android/pull/77/files#diff-01dafb3fd0217441330103cfc8300094)

(We totally understand if you are too busy or not available. No problem, and thank you for your past contributions.)

Best regards,
\- DeeDeeG

**Comment by Carmen-Git-It** [comment]
It looks good to me technically! (I have not tested yet, but it doesnt seem prone to issues the way it was implemented) Really appreciate the work put into making the strings generic, sorry you had to do all that extra work just to add your translations!

That being said I'm unable to evaluate the translations.

File: app/src/main/res/values-es-rUS/strings.xml (hunk 1 above)
Comment (thehme) [reply app/src/main/res/values-es-rUS/strings.xml]: Sounds good, I can revert this.

Comment (thehme) [reply app/src/main/res/values-es-rUS/strings.xml]: Hi @DeeDeeG, I removed the app name translation as you suggested, so I think this is ready when you want to merge.

**Comment by DeeDeeG** [comment]
@thehme, Thanks for the changes.

I defer to @hissingpanda or @Carmen-Git-It for merging, since they are the Android experts. I just try to keep communication co-ordinated and everyone in the loop.

We could potentially merge as-is and treat reviewing the actual translation strings as a follow-up task, I suppose. It is sometimes hard to find a reviewer to look over translations on any sort of short notice. These translations look reasonable to me, but I'm not a fluent Spanish speaker, so my review isn't so meaningful.

**Comment by Intimaria** [comment]
Hi, I can take a look on the weekend. Thanks for the heads up.

**Comment by hissingpanda** [comment]
Deployed this to the play store today. should be available in a few hours.

**Comment by Intimaria** [comment]
> If you would like to help review the translations, you can see them here: [`values-es-rUS/strings.xml`](https://github.com/RefugeRestrooms/refugerestrooms-android/pull/77/files#diff-2b5d718e97db44f62e321bf20c2363d2)
> 
I only have some gentle nudges around the spelling of words: 
line 22: should read "Añadir baño"
lines 37: should read "Nunca mostrar esto otra vez" or "No mostrar esto otra vez".

**Comment by thehme** [comment]
@Intimaria will check if it still needs fixing

**Comment by DeeDeeG** [comment]
Hi all. The suggestions by @Intimaria look good to me (having checked in with Google Translate and [SpanishDict.com](https://www.spanishdict.com/conjugate/mostrar)). @thehme would you be interested in adding these as a second, tiny Pull Request?

**Comment by thehme** [comment]
@DeeDeeG yes, will do

**Comment by thehme** [comment]
@DeeDeeG made the two typo fixes and also another one i found. PR at

https://github.com/RefugeRestrooms/refugerestrooms-android/pull/80

**Pull request merged**